app/models/concerns/user_authentication.rb

```diff
@@ -9,11 +9,12 @@ module UserAuthentication
 
   class_methods do
     def available_omniauth_providers
-      providers = [:tunnistamo]
-      providers << :suomifi if Decidim::Suomifi.configured?
-      providers << :mpassid if Decidim::Mpassid.configured?
-
-      providers
+      []
+      #providers = [:tunnistamo]
+      #providers << :suomifi if Decidim::Suomifi.configured?
```

Comment:
I'm not sure why these blocks seem to be getting configured even though config/environments/development.rb has them turned off:

```
  config.suomifi_enabled = false
  config.mpassid_enabled = false
```

In config/initializers/suomifi.rb, it appears to check those application config values before configuring those modules:

```
if Rails.application.config.suomifi_enabled
...
```

Anyway, I commented this out rather than removing it, but can be persuaded to remove it. I think at some point in following days we need a task in trello to completely strip out Mpassid, Suomifi and Tunnistamo.